### PR TITLE
Added Ebay Inc's(EBAY) data

### PR DESCRIPTION
**Pull request by Dawny33**

### CODE REVIEW

File: test/readwrite.jl [hunk 1]
```diff
@@ -42,5 +43,6 @@ facts("const objects have expected length") do
         @fact length(BA)   --> 13090
         @fact length(CAT)  --> 13090
         @fact length(DELL) --> 6389
+        @fact length(EBAY) --> 4383
```
Comment (milktrader) [inline test/readwrite.jl]: Failed tests have incorrect values
````
const objects have expected length: .........FF
  Failure :: (line:-1) :: fact was false
    Expression: length(DELL) --> 6389
      Expected: 6389
      Occurred: 6388
  Failure :: (line:-1) :: fact was false
    Expression: length(EBAY) --> 4383
      Expected: 4383
      Occurred: 4382
````

**Comment by Dawny33** [comment]
@milktrader Pl review

**Comment by milktrader** [comment]
The unit tests were still off by one for the data set lengths, but I fixed that. Before I tag this, why does DELL data stop at the end of October 2013 instead of December?

**Comment by Dawny33** [comment]
Not sure why the unit tests are failing!  They ran alright on my PC.

Yeah, there is no `Nov '13` and `Dec '13` in the Dell's dataset.

**Pull request merged**